pyproject.toml

```diff
@@ -47,6 +40,46 @@ tests = [
     "pytest >=6.0",
     "pytest-dotenv",
     "requests-mock",
+    "apache-airflow-providers-google",
+    "sqlalchemy-bigquery==1.3.0",
+    "smart-open[all]>=5.2.1",
+    "apache-airflow-providers-snowflake",
+    "snowflake-sqlalchemy ==1.2.0",
+    "snowflake-connector-python[pandas]",
+    "apache-airflow-providers-postgres",
+    "s3fs",
+    "smart-open[s3]>=5.2.1",
+]
+gcp = [
+    "apache-airflow-providers-google",
+    "sqlalchemy-bigquery==1.3.0",
+    "smart-open[gcs]>=5.2.1",
+]
+snowflake = [
+    "apache-airflow-providers-snowflake",
+    "snowflake-sqlalchemy ==1.2.0",
+    "snowflake-connector-python[pandas]",
+]
+postgres = [
+    "apache-airflow-providers-postgres",
+]
+aws = [
+    "s3fs",
+    "smart-open[s3]>=5.2.1",
+]
+all = [
```

Comment:
@uranusjr Does `all` keyword work with flit -- https://github.com/pypa/flit/issues/488

I wanted to use `flit` for https://github.com/astronomer/astronomer-providers/ but the following two issues made me think otherwise:

- https://github.com/pypa/flit/issues/525 (Minor, yet annoying)
- https://github.com/pypa/flit/issues/488

Reply:
The keys here can be arbitrary, everything should work. (Well, if it is a valid extra name.)

Reply:
Flit currently only puts license information in trove classifiers. The license field is pending standardisation. (If we really want to we can probably extend Flit to add that field to the wheel, but the value wouldn’t be very useful due to its lack of standardisation.)

The second issue you linked is requesting a “magic” extra that’s added automatically without declaring. IMO that’s not that good an idea, but manually declaring `all` should work just fine.

Reply:
Thank you very much, @kaxil & @uranusjr , I always learn a lot from your comments!